### ISSUE TITLE
Chips: ARIA/Keyboard navigation trigger itemClick event with 'Space' key missing

### ISSUE THREAD

**Issue by f94e59** [primefaces/primeng#15611]
### Describe the bug

Expected:

1. You can change focus state of items with arrow keys
2. If an item is focused you can trigger the itemClick emitter with the 'Space' key

Given:

1. You can change focus state of items with arrow keys
2. There is currently no option to trigger itemClick emitter with the keyboard navigation

### Environment

PrimeNG: 17.16.1
Angular: 17.2.0

### Angular version

17.2.0

### PrimeNG version

17.16.1

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

N/A

### Steps to reproduce the behavior

1. You can change focus state of items with arrow keys
2. There is currently no option to trigger itemClick emitter with the keyboard navigation

### Expected behavior

1. You can change focus state of items with arrow keys
2. If an item is focused you can trigger the itemClick emitter with the 'Space' key
